src/tests/end2end/CopyTests.cpp

```diff
@@ -84,18 +85,23 @@ class CopyTests_T2B : public CopyTests {
             uint32_t height = textureSpec.height >> textureSpec.level;
             uint32_t rowPitch = Align(kBytesPerTexel * width, kTextureRowPitchAlignment);
             uint32_t texelsPerRow = rowPitch / kBytesPerTexel;
-            uint32_t texelCount = texelsPerRow * (height - 1) + width;
+            uint32_t texelCountPerLayer = texelsPerRow * (height - 1) + width;
 
             // Create an upload buffer and use it to populate the `level` mip of the texture
```

Comment:
nit: fix comment to refer to multiple buffers.

Alternatively we could have a single buffer with all the layers and then do copies starting at an offset that depends on the layer.

Reply:
Done